docs/reference/buildx_prune.md

```diff
@@ -21,6 +21,26 @@ Remove build cache
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Clears the build cache of the selected builder.
+
+You can finely control what cache data is kept using:
+
+- The `--filter=until=<duration>` flag to keep images that have been used in
+  the last `<duration>` time.
+
+  `<duration>` is a duration string, e.g. `24h` or `2h30m`, with allowable
+  units of `(h)ours`, `(m)inutes` and `(s)econds`.
+
+- The `--keep-storage=<space>` flag to keep `<space>` bytes of data in the cache.
```

Comment:
Just a quick thought; perhaps `<size>` would be a good alternative, as we already use "size" in various places